packages/core/src/components/ListContainer/ListItem/ListItem.tsx

```diff
@@ -176,54 +184,48 @@ export const HvListItem = (props: HvListItemProps) => {
       : {};
 
   const listItem = (
-    <StyledListItem
+    <li
       id={id}
       role={role}
       value={value}
-      onClick={handleOnClick}
-      onKeyDown={() => {}}
-      className={clsx(
-        className,
-        listItemClasses.root,
-        classes?.root,
-        !disableGutters && clsx(listItemClasses.gutters, classes?.gutters),
-        condensed && clsx(listItemClasses.condensed, classes?.condensed),
-        interactive && clsx(listItemClasses.interactive, classes?.interactive),
-        selected && clsx(listItemClasses.selected, classes?.selected),
-        disabled && clsx(listItemClasses.disabled, classes?.disabled),
-        startAdornment != null &&
-          clsx(listItemClasses.withStartAdornment, classes?.withStartAdornment),
-        endAdornment != null &&
-          clsx(listItemClasses.withEndAdornment, classes?.withEndAdornment)
+      className={cx(
+        classes.root,
+        {
+          [classes.gutters]: !disableGutters,
+          [classes.condensed]: condensed,
+          [classes.interactive]: interactive,
+          [classes.selected]: !!selected,
+          [classes.disabled]: !!disabled,
+          [classes.withStartAdornment]: startAdornment != null,
+          [classes.withEndAdornment]: endAdornment != null,
+        },
+        className
       )}
-      $gutters={!disableGutters}
-      $interactive={interactive}
-      $disabled={disabled || false}
-      $selected={selected || false}
-      $startAdornment={startAdornment != null}
-      $endAdornment={endAdornment != null}
+      {...(!!interactive && {
+        onClick: handleOnClick,
+      })}
```

Comment:
I'm not sure about making this change now... `HvListItem` is one of those components that are used everywhere, someone might be using `onClick` without `interactive`...

Reply:
I'll change this and put it back as it was since we don't know what other people are doing

Reply:
Updated 🚀